quotes.json

```diff
@@ -4953,6 +4953,10 @@
     "author": "Rachel Carson",
     "text": "If facts are the seeds that later produce knowledge and wisdom, then the emotions and the impressions of the senses are the fertile soil in which the seeds must grow."
   },
+  {
+    "author":"Rahul Dravid",
+    "text": "You don't play for revenge. You play for respect and pride"
```

Comment:
Hi @moksh-shukla, 
Thanks for creating this Pull Request. 👍 
Do you have a _source_ for this quote? 🤷 
We can't find this on https://en.wikiquote.org/wiki/Rahul_Dravid (_his most famous quotes_)
We are very happy to accept a quote from one of India's _finest_, but it must have a source to avoid people adding random things that may have been _attributed_ but not actual quotes. 
<!--
Sadly, instagram https://www.instagram.com/p/BchG17WA27z  is not a _reliable_ source; there's a lot of fake stuff  on there. 🙄 
Even though this post has a few "likes" it still doesn't make it a source. 
![image](https://user-images.githubusercontent.com/194400/94790179-d9ab6700-03cd-11eb-8eca-f9c4a85e419a.png)
-->

Reply:
Hi @nelsonic ,
Thanks for the review. I will add the source link to the quote.